test/modules/paymentProcessor/bridging/abstracts/CrosschainBase_v1_Exposed.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: LGPL-3.0-only
```

Comment:
Should be in `test/modules/paymentProcessor/abstracts/CrosschainBase_v1_Exposed.sol` same folder as the test file